src/wp-includes/class-wp-script-modules.php

```diff
@@ -225,6 +226,35 @@ public function print_import_map() {
 		}
 	}
 
+	/**
+	 * Prints the necessary script to load import map polyfill for browsers that
+	 * do not support import maps.
+	 *
+	 * @since 6.5.0
+	 */
+	public function print_import_map_polyfill() {
+		$import_map = $this->get_import_map();
+		if ( ! empty( $import_map['imports'] ) ) {
+			$test = 'HTMLScriptElement.supports && HTMLScriptElement.supports("importmap")';
+			$src  = includes_url( 'js/dist/vendor/wp-polyfill-importmap.min.js' );
```

Comment:
The version number is not passed with the URL so it might create browser cache issues when the related packages gets updated in the future.